tests/test_client_functional.py

```diff
@@ -54,6 +54,36 @@ async def handler(request):
     assert 1 == len(client._session.connector._conns)
 
 
+async def test_keepalive_after_head_requests_success(
+        aiohttp_client) -> None:
+    async def handler(request):
+        body = await request.read()
+        assert b'' == body
+        return web.Response(body=b'OK')
+
+    cnt_conn_reuse = 0
+    async def on_reuseconn(session, ctx, params):
```

Comment:
```suggestion
    cnt_conn_reuse = 0

    async def on_reuseconn(session, ctx, params):
```
This change fixes the linter.
The PR should have green checkers